sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -1444,6 +1462,40 @@ void CNPC_Combine::GatherConditions()
 		}
 	}
 #endif
+
+#ifdef EZ2
+	ClearCondition( COND_COMBINE_CAN_ORDER_SURRENDER );
+
+	if (HasCondition(COND_CAN_RANGE_ATTACK1))
+	{
+		if (CanOrderSurrender())
+		{
+			if ( GetEnemy() /*&& GetEnemies()->NumEnemies() <= 2*/ && GetEnemy()->ClassMatches("npc_citizen") && GetEnemy()->MyNPCPointer()->GetActiveWeapon() == NULL )
+			{
+				CNPC_Citizen *pCitizen = static_cast<CNPC_Citizen*>(GetEnemy());
+				if (pCitizen && pCitizen->CanSurrender())
+				{
+					// Make sure they're not about to pick up a weapon
+					if (!pCitizen->IsCurSchedule( SCHED_NEW_WEAPON, false ))
+					{
+						// Finally, check if the citizen has spoken beg or surrender concepts already
+						float flTimeSpeakSurrender = pCitizen->GetTimeSpokeConcept( TLK_SURRENDER );
+						float flTimeSpeakBeg = pCitizen->GetTimeSpokeConcept( TLK_BEG );
+						if ((flTimeSpeakSurrender != -1 && gpGlobals->curtime - flTimeSpeakSurrender >= 2.0f) || (flTimeSpeakBeg != -1 && gpGlobals->curtime - flTimeSpeakBeg >= 2.0f))
```

Comment:
Maybe this value (2.0f seconds) should be configurable? I would also suggest that it should be lessened for "beg". Because citizens speak the surrender concept right away and the beg concept later, "beg" implies they have been cowering for slightly longer. I think you could drop it down to 0.5 or 1.0.

This is only a suggestion